src/solvers/dgsem_tree/dg_1d.jl

```diff
@@ -44,10 +44,15 @@ function create_cache(mesh::TreeMesh{1}, nonconservative_terms::Val{false}, equa
   NamedTuple()
 end
 
-# TODO: MHD in 1D
```

Comment:
More a comment than actual code. The divergence-free constraint is automatically satisfied for 1D MHD such that the non-conservative term (and the GLM variable) is unneeded. 

So I think this should get updated once we have a different set of 1D equations (like shallow water) that really needs non-conservative terms.

Reply:
Yeah, that was my original motivation. @phioeffn told me that he wants to use/implement shallow water in 1D with some students, so I prepared the necessary basic setup to help them getting started.